WindowsServerDocs/remote/remote-desktop-services/rds-vdi-recommendations-2004.md

```diff
@@ -0,0 +1,794 @@
+  - Changes such as event log writes, log writes, etc. are redirected to the read/write virtual disk assigned to that VM.
+  - In this circumstance, OS and app servicing may operate normally, using traditional servicing software such as Windows Server Update Services, or other management technologies.
+- The difference between a persistent virtual desktop device and a “normal” virtual desktop device” is the relationship to the master/gold image. At some point updates must be applied to the master. It is at this point where organizations decide how the user persistent changes are handled. In some cases, the disk with the user changes is discarded and/or reset.  It may also be that the changes the user makes to the machine are kept through monthly Quality Updates, and the base is reset following a Feature Update.
+
+### Non-Persistent Virtual Desktop Environments
+
+When a non-persistent virtual desktop implementation is based on a base or “gold” image, the optimizations are mostly performed in the base image, and then through local settings and local policies.
+
+With image-based non-persistent (NP) virtual desktop environments, the base image is read-only. When a NP virtual desktop device (VM) is started, a copy of the base image is streamed to the VM.  Activity that occurs during startup and thereafter until the next reboot is redirected to a temporary location. Usually the users are provided network locations to store their data.  In some cases, the user’s profile is merged with the standard VM to provide the user their settings.
+
+One important aspect of NP virtual desktop that is based on a single image, is servicing.  Updates to the operating system (OS) and components of the OS are delivered usually once per month.  With image based virtual desktop environment, there are a set of processes that must be performed to get updates to the image:
+
+- On a given host, all the VMs on that host, based from the base image must be shut down / turned off. This means the users are redirected to other VMs.
+	-In some implementations, this is referred to as “draining”.  The virtual machine or session host, when set to draining mode, stops accepting new requests, but continues servicing users currently connected to the device.
+	-In draining mode, when the last user logs off the device, that device is then ready for servicing operations.
+-The base image is then opened and started up.  All maintenance activities are then performed, such as OS updates, .NET updates, app updates, etc.
+-Any new settings that need to be applied are applied at this time.
+-Any other maintenance is performed at this time.
+-The base image is then shut down.
+-The base image is sealed and set to go back into production.
+-Users are allowed to log back on.
```

Comment:
Are these dashes put like this intentionally, as to not create a bullet point list?
If a bullet point list was intended, please add the required spacing.

Reply:
No this was supposed to be a bullet list and has been fixed.